src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -195,7 +195,7 @@ public Visitable visit(MethodDeclaration method, Void p) {
       targetMethods.add(method.resolve().getQualifiedSignature());
       unfoundMethods.remove(methodName);
       Type returnType = method.getType();
-      if (returnType instanceof ReferenceType) {
+      if (returnType.resolve() instanceof ResolvedReferenceType) {
```

Comment:
Please add a comment here explaining that this is working around a JavaParser bug. (The comment should explain exactly what you explained in the comment you opened on this PR.)